molgenis-app/pom.xml

```diff
@@ -52,6 +52,23 @@
 						<artifactId>mysql-connector-java</artifactId>
 						<version>${mysql.connector.java.version}</version>
 					</dependency>
+					<dependency>
+						<groupId>uk.ac.ebi.mydas</groupId>
+						<artifactId>mydas</artifactId>
+						<version>1.6.7</version>
+						<scope>system</scope>
+						<systemPath>${basedir}/../molgenis-das/src/main/resources/mydas-1.6.7.jar</systemPath>
+					</dependency>
+					<dependency>
+						<groupId>xpp3</groupId>
+						<artifactId>xpp3</artifactId>
+						<version>1.1.3.4.O</version>
+					</dependency>
+					<dependency>
```

Comment:
The problem is, is that tomcat uses the created WAR, which contains all the correct dependencies.

Jetty does not use the war, but some fancy regex to collect all the dependencies. MyDas is not included in this dependency grab. This is why I added it here explicitly